src/server/cloak-server.js

```diff
@@ -1,9 +1,27 @@
 var cloak = require('cloak');
 
+var listOfUsers = [];
+
+var sendLobbyCount = function(arg) {
```

Comment:
Whenever a user exits or enters the lobby this function is called and then updates the list of users. It would make more sense if we renamed the function "updateUserList".